concepticondata/conceptlists/Valenzuela-2023-44.tsv

```diff
@@ -0,0 +1,45 @@
+ID	NUMBER	NUMBER_IN_TABLE	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	TABLE
+Valenzuela-2023-44-1	1	1	Tree	906	TREE	Lexical_List
```

Comment:
Please keep the Table entries as we had discussed them: "lexical", "body-parts", and "morphological". We do not want the underscores and upper case in those entries.